### ISSUE TITLE
gem dependencies not loading in main app

### ISSUE THREAD

**Issue by saturnflyer** [jayroh/thredded#116]
In my project at https://github.com/saturnflyer/purple_door/tree/forum I had to add references to gems which are dependencies of thredded.
Here's the relevant part of the Gemfile. I got sass errors without adding the 3 dependencies in the main app.

```ruby
gem 'thredded', git: 'https://github.com/jayroh/thredded.git', ref: '39da38c744ab5eee4c421c64e64c1e557b7ea06c'
gem 'bourbon', '4.2.3'
gem 'neat', '1.7.2'
gem 'bitters', '1.0.0'
```

Additionally, the application would not compile the assets unless I added this to my `config/initializers/assets.rb` file:

```ruby

Rails.application.config.assets.precompile += %w( chosen-sprite.png chosen-sprite@2x.png breadcrumb-cheveron.svg )
```

**Comment by jayroh** [comment]
@saturnflyer - Interesting. I think those gems are added when a theme is installed but, ideally, it *should* work without those gems! I'll take a look!

**Comment by jayroh** [comment]
fixed with #127
